doc_source/s3-encryption-client.rst

```diff
@@ -16,20 +16,25 @@
    :description: Client-side encryption for the Amazon S3 client with the AWS SDK for PHP version 3.
    :keywords: AWS SDK for PHP version 3 constructor, AWS SDK for PHP version 3 client configuration
 
-The |sdk-php| provides an ``S3EncryptionClientV2``. With client-side
-encryption, data is encrypted and decrypted directly in your environment. This
+With client-side encryption, data is encrypted and decrypted directly in your environment. This
 means that this data is encrypted before it's transferred to |S3|, and you
 don’t rely on an external service to handle encryption for you. For new implementations,
-we suggest the use of ``S3EncryptionClientV2`` over the deprecated ``S3EncryptionClient``.
-Older implementations still using ``S3EncryptionClient`` should attempt to migrate.
-``S3EncryptionClientV2`` maintains support for decrypting data that was encrypted
-using the legacy ``S3EncryptionClient``.
+we suggest the use of ``S3EncryptionClientV2`` and ``S3EncryptionMultipartUploaderV2`` over the deprecated
+``S3EncryptionClient`` and ``S3EncryptionMultipartUploader``.  It is recommended that older implementations
+still using the deprecated versions attempt to migrate. ``S3EncryptionClientV2`` maintains
+support for decrypting data that was encrypted using the legacy ``S3EncryptionClient``.
 
 The |sdk-php| implements :KMS-dg:`envelope encryption <workflow>`
 and uses `OpenSSL <https://www.openssl.org/>`_ for its encrypting and
 decrypting. The implementation is interoperable with :AWS-gr:`other SDKs that match its feature support <aws_sdk_cryptography>`.
 It's also compatible with :doc:`the SDK’s promise-based asynchronous workflow <guide_promises>`.
 
+Migration Guide
+==========
+
+For those who are trying to migrate to from the deprecated clients to the new clients, there is a migration
+guide which can be found at https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/s3-encryption-migration.html

```

Comment:
I believe this should be changed to the proper RST link format, and perhaps changed to something like `can be found here`, with `here` as the link.